addon/components/polaris-icon.hbs

```diff
@@ -0,0 +1,25 @@
+<span
```

Comment:
Out of curiosity, why isn't this in `addon/templates/components`? 🤔

Reply:
This co-locates the template and component file. Change done through [codemod](https://github.com/ember-codemods/ember-component-template-colocation-migrator) & [RFC here](https://github.com/emberjs/rfcs/pull/481).
This will be needed for all components in order to support Octane-only consuming apps.

Btw, sorry, wanted to, but completely forgot to mention this in the description.

PS: These can also be [nested](https://github.com/ember-codemods/ember-component-template-colocation-migrator#nested-component-structure), but for now went with the default mode. We can change later probably if we feel it's better for us